lib/coverage.js

```diff
@@ -32,8 +32,37 @@ function Coverage(config) {
             return;
         }
 
+        function doesOneRegexMatch(regexes, value) {
+            var results = regexes.map(function(regex) {
+                try {
```

Comment:
This try/catch can be removed if the `karmaTypescriptConfig.coverageOptions.exclude` is validated on startup in [configuration.js](https://github.com/monounity/karma-typescript/blob/2.1.8/lib/configuration.js#L68)